projects/js-packages/publicize-components/src/components/post-publish-share-status/share-status.tsx

```diff
@@ -6,15 +6,23 @@ import Notice from '../notice';
 import { ShareStatusModalTrigger } from '../share-status';
 import styles from './styles.module.scss';
 
+type ShareStatusProps = {
+	reShareTimestamp?: number | null;
+};
+
 /**
  * Share status component.
  *
- *
- * @return {import('react').ReactNode} - Share status UI.
+ * @param {ShareStatusProps} props - component props
+ * @return {import('react').ReactNode} - React element
  */
-export function ShareStatus() {
+export function ShareStatus( { reShareTimestamp = null }: ShareStatusProps ) {
 	const shareStatus = useSelect( select => select( socialStore ).getPostShareStatus(), [] );
 
+	const currentShares = reShareTimestamp
+		? shareStatus.shares.filter( share => share.timestamp * 1000 > reShareTimestamp )
```

Comment:
Why do we need to multiply by 1000 here?

Reply:
Because of the milliseconds, we had to do it here too https://github.com/Automattic/jetpack/blob/7a5af0f7b1ef3a83e73b89eff2831ebe2a0bdea9/projects/js-packages/publicize-components/src/components/share-status/shares-dataview.tsx#L58

Reply:
That other one is for displaying the value but comparison should be between the same unit and we have our timestamps in seconds not milliseconds.

Reply:
Date uses milliseconds so we either have to multiple or divide by 1000

Reply:
I'm not sure I understand this. The timestamp on the sharesData is in seconds. Date.now gives milliseconds. I multipled the earlier so it's the same unit (milliseconds).

Reply:
You can see the data on the picture I attached

Reply:
Ah sorry. I read that as we are multiplying the share item timestamp. Thank you for clarifying.